reversion_compare/mixins.py

```diff
@@ -223,10 +223,12 @@ def compare_DateTimeField(self, obj_compare):
         return render_to_string("reversion-compare/compare_DateTimeField.html", context)
 
     def compare_BooleanField(self, obj_compare):
-        ''' compare all model BooleanField field '''
+        ''' compare booleans as a complete field, rather than as a string '''
         context = {
-            "bool1": obj_compare.value1,
-            "bool2": obj_compare.value2,
+            "bool1": str(obj_compare.value1),
+            "bool2": str(obj_compare.value2),
```

Comment:
No its not, I've previously had troubles with None in templates, but this isn't an issue now.